frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -503,8 +491,78 @@ function RunAiModelTab({ aIJobModalState }: { aIJobModalState: string }) {
   );
 }
 
+function AlignmentTab() {
+  const centerImageStyle = {
+    margin: "auto",
+    width: 150,
+  };
+  return (
+    <div>
+      <Space align="center">
+        <Radio.Button className="aIJobSelection" checked={true}>
+          <Card bordered={false}>
+            <Space direction="vertical" size="small">
+              <Row className="ai-job-title">Align Sections</Row>
+              <Row>
+                <img
+                  src={`/assets/images/${jobNameToImagePath.align_sections}`}
+                  alt={"Example of improved alignment of slices"}
+                  style={centerImageStyle}
+                />
+              </Row>
+            </Space>
+          </Card>
+        </Radio.Button>
+      </Space>
+      <AlignSectionsForm />
+    </div>
+  );
+}
+
+function UseSkeletonFormItem() {
+  const tracing = useSelector((state: OxalisState) => state.tracing);
+  return (
+    <div>
+      <Form.Item
+        name="useAnnotation"
+        label={
+          <Space>
+            <div style={{}}>
+              Manual Matches
+              <Tooltip title="Please select whether the alignment should take connected skeleton nodes between adjacent sections as alignment guideline whenever available.">
+                <InfoCircleOutlined />
+              </Tooltip>
+            </div>
+          </Space>
+        }
+        valuePropName="checked"
+        rules={[
+          {
+            validator: (_rule, checked) => {
+              if (checked) {
+                if (tracing.annotationId === "") {
+                  return Promise.reject(
+                    "No annotation was found. Please create an annotation first.",
+                  );
+                }
+                if (tracing.skeleton == null) {
+                  return Promise.reject("No skeleton was found. Please create a skeleton first.");
+                }
+              }
+              return Promise.resolve();
+            },
+          },
```

Comment:
Just a suggestion: Maybe also include a check for whether the skeleton tracing has at lease one tree? (Is not empty)